lib/transport-common.js

```diff
@@ -29,7 +31,7 @@ const common = require('./common');
 const newConnectFn = (
   connFactory, transportClass
 ) => (app, client, ...options) => {
-  const callback = common.extractCallback(options);
+  const callback = common.cbExtract(options);
```

Comment:
In this case not providing a callback is both an error and strange (you are calling a function to create a connection and not expecting a result), that's why I used extract here.